### PR TITLE
docs(meshmetric): adjust table and rename usedOnly

### PR DESCRIPTION
**Pull request by slonka**
adjust targetting table and rename usedOnly to includeUnused

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: app/_src/policies/meshmetric.md [hunk 1]
```diff
@@ -30,13 +30,33 @@ If you haven't already read the [observability docs](/docs/{{ page.version }}/ex
...
+{% tab targetRef Sidecar %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`, `MeshSubset`, `MeshService`, `MeshServiceSubset` |
+{% endtab %}
+
+{% tab targetRef Builtin Gateway %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`                                                   |
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:44]: not sure here for the gateway, FYI @Automaat 

validation def won't allow it https://github.com/kumahq/kuma/blob/df79d3a77cae0d72fddd4a7bc87857ab4db8c486/pkg/plugins/policies/meshmetric/api/v1alpha1/validator.go#L24

File: app/_src/production/mesh.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ spec:
 
 {{site.mesh_product_name}} consumes all [Policies](/policies) on the cluster and joins each to an individual `Mesh`, identified by this property.
 {% endtab %}
-{% tab policies Kubernetes tab (New policies) %}
+{% tab policies Kubernetes (New policies) %}
```
Comment (slonka) [inline app/_src/production/mesh.md:126]: not related to this, just noticed an extra "tab" from my previous PR

File: app/_src/policies/meshmetric.md [hunk 3]
```diff
@@ -30,19 +30,31 @@ If you haven't already read the [observability docs](/docs/{{ page.version }}/ex
...
+| `targetRef.kind`        | `Mesh`, `MeshSubset`, `MeshService`, `MeshServiceSubset` |
 {% endtab %}
 
 {% tab targetRef Builtin Gateway %}
-| `targetRef`             | Allowed kinds                                             |
-| ----------------------- | --------------------------------------------------------- |
-| `targetRef.kind`        | `Mesh`, `MeshGateway`, `MeshGateway` with listener `tags` |
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`                                                   |
```
Comment (Automaat) [inline app/_src/policies/meshmetric.md:44]:
```suggestion
| `targetRef.kind`        | `Mesh`, `MeshGateway`                     |
```

File: app/_src/policies/meshmetric.md (hunk 1 above)
Comment (Automaat) [reply app/_src/policies/meshmetric.md:44]: changed, now we support `MeshGateway` targetRef

File: app/_src/policies/meshmetric.md [hunk 4]
```diff
@@ -43,6 +44,17 @@ If you haven't already read the [observability docs](/docs/{{ page.version }}/ex
 | `targetRef.kind`        | `Mesh`, `MeshGateway`, `MeshGateway` with listener `tags` |
 {% endtab %}
 {% endtabs %}
+{% endif_version %}
+{% if_version lte:2.5.x %}
```
Comment (michaelbeaumont) [inline app/_src/policies/meshmetric.md:48]: Did we have MeshMetric before 2.6?

**Review by Automaat — APPROVED** [review]

**Pull request merged**